kafka/src/main/java/org/apache/apex/malhar/kafka/KafkaConsumerWrapper.java

```diff
@@ -129,6 +130,11 @@ public void emitImmediately(Map<AbstractKafkaPartitioner.PartitionMeta, Pair<Lon
         if (meta.getTopicPartition().equals(tp)) {
           kc.resume(tp);
         } else {
+          try {
```

Comment:
Why do we need to catch the exception here? It will always reposition some partition to the beginning which is not correct